packages/api/tests/task.test.js

```diff
@@ -2060,17 +2009,14 @@ describe('Task tests', () => {
         promisedField: [{ location_id }],
       });
 
-      patchTaskDateRequest(
-        { user_id, farm_id },
-        patchTaskDateBody,
-        task.task_id,
-        async (err, res) => {
-          expect(res.status).toBe(200);
-          const updated_task = await getTask(task_id);
-          expect(updated_task.due_date).toBe(due_date);
-          done();
-        },
-      );
+      patchTaskDateRequest({ user_id, farm_id }, patchTaskDateBody, task_id, async (err, res) => {
+        expect(res.status).toBe(200);
+        const updated_task = await getTask(task_id);
+        expect(updated_task.due_date.toISOString().split('T')[0]).toBe(due_date);
+        done();
+      });
     });
   });
 });
+
+/* global jest describe test xtest expect beforeEach afterAll */
```

Comment:
@MwayaB This block comment that starts with `global` is actually a configuration instruction to the eslint tool. JavaScript doesn't have the equivalent of C/C++ header files that you can include to tell the tools that something is defined somewhere else.
This tells eslint that the listed names are defined globally (outside this file), so the tool should not complain that they aren't defined here. Lots of the test files are missing this sort of comment, and the better way is to put equivalent information in a central config file, but this was convenient for now.
My guess is that you may have `require`d (the wrong) `test` at the top of this file because eslint was complaining about it.